miner/worker.go

```diff
@@ -1324,6 +1324,10 @@ func (w *worker) computeBundleGas(bundle types.Transactions, parent *types.Block
 		if err != nil {
 			return simulatedBundle{}, err
 		}
+		if receipt.Status == types.ReceiptStatusFailed {
```

Comment:
this failure status may happen also in other cases:
* OutOfGasException (normal case)
* StackOverflow (this would be for corrupted bytecode mostly)
* StackUnderflow (this would be for corrupted bytecode mostly)
* BadOpcode (this would be for corrupted bytecode mostly)
* PrecompileFialure (only for direct calls to precompiles)

I. believe it is fine to leave the code the way it is but it may be misleading to add revert as a reason for each of those